build.gradle

```diff
@@ -150,10 +152,20 @@ task assembleProtos {
     dependsOn 'generateFlatbuffersCpp'
 }
 
+
+String regex = /(?i)rlbot(?i)[-_ ](?i)core(?i)/
+new File(parentFolder).eachDir{
+    if (it.name ==~ regex) {
+        defaultCoreFolder = it.absolutePath
```

Comment:
If there are multiple folders which match the regex, one of them will be chosen arbitrarily. If you were going to document this, you'd have to say something like "it'll copy from the last folder that matches XYZ, with 'last' defined however eachDir feels like".

Rather than risk that aggravation, I would prefer to skip the regex, add a decent error message, and then let people override the default rlbot-core folder in local.properties.

Reply:
I would rather not deal with needing to maintain a local.properties file for something as silly as this.

This is already a very niche use case
I have added an error that is thrown if more than one folder exist